.github/workflows/cd.yaml

```diff
@@ -32,45 +66,76 @@ jobs:
         with:
           name: ${{ env.OUTPUT }}
           path: ${{ env.OUTPUT }}
+      - name: Upload to continuous release
+        run: |
+          gothub upload \
+            --user "$(echo ${{ github.repository }} | cut -d/ -f1)" \
+            --repo "$(echo ${{ github.repository }} | cut -d/ -f2)" \
+            --tag continuous \
+            --replace \
+            --name ${{ env.OUTPUT }} \
+            --file ${{ env.OUTPUT }}
 
   appimage_x86_64:
     runs-on: ubuntu-16.04
+    needs: setup
     env:
+      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
       ARCH: x86_64
-      OUTPUT: endless-sky-x86_64-${{ github.sha }}.AppImage
+      OUTPUT: endless-sky-x86_64-continuous.AppImage
     steps:
       - uses: actions/checkout@v2
+      - name: Install gothub
+        run: |
+          go get github.com/itchio/gothub
+          echo "::set-env name=GOPATH::$(go env GOPATH)"
+          echo "::add-path::$(go env GOPATH)/bin"
       - name: Install dependencies
         run: |
           sudo rm /etc/apt/sources.list.d/* && sudo dpkg --clear-avail # Speed up installation and get rid of unwanted lists
           sudo apt-get update
           sudo apt-get install -y --no-install-recommends libsdl2-dev libpng-dev libjpeg-turbo8-dev libopenal-dev libmad0-dev libglew-dev libgl1-mesa-dev libegl1-mesa-dev libgles2-mesa-dev scons
+      - name: Adjust version strings
+        run: ./utils/cd_version_strings.sh
       - name: Build AppImage
         run: ./utils/build_appimage.sh
       - name: Upload artifact
         uses: actions/upload-artifact@v1
         with:
           name: ${{ env.OUTPUT }}
           path: ${{ env.OUTPUT }}
+      - name: Upload to continuous release
+        run: |
+          gothub upload \
+            --user "$(echo ${{ github.repository }} | cut -d/ -f1)" \
+            --repo "$(echo ${{ github.repository }} | cut -d/ -f2)" \
+            --tag continuous \
+            --replace \
+            --name ${{ env.OUTPUT }} \
+            --file ${{ env.OUTPUT }}
 
   windows_win64:
     runs-on: windows-latest
+    needs: setup
     env:
+      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
       DIR_MINGW64: C:\ProgramData\chocolatey\lib\mingw\tools\install\mingw64\x86_64-w64-mingw32
       DIR_ESLIB: .\dev64
-      OUTPUT: EndlessSky-win64-${{ github.sha }}.zip
+      OUTPUT: EndlessSky-win64-continuos.zip
```

Comment:
Shouldn't this be `      OUTPUT: EndlessSky-win64-continuous.zip` ? (One missing "u" in "continuous")

Reply:
whoops. good spot.

Reply:
You've also got the same spelling mistake on your first post. I was just going to comment on that, then decided to take a look through the rest just in case.